docs/README.md

```diff
@@ -158,22 +158,4 @@ termpair share --cmd "gdb -p 1234"
 
 # Contributing
 
-Clone repo, then run
-
-```
-pip install -e .
-```
-
-to modify Python code.
-
-To modify frontend code, run:
-
-```
-make build_frontend
-```
-
-Then reload the TermPair server with:
-
-```
-termpair server
-```
+Interested in contributing to TermPair? You'll find all the information you need in our [Contributing guidelines](https://github.com/cs01/termpair/blob/master/CONTRIBUTING.md).
```

Comment:
This is fine for now but long term, as more pages are added, it would be nice to move to mkdocs' navigation:

https://www.mkdocs.org/user-guide/configuration/#nav

Reply:
Do you mean, use mkdocs' navigation to include the `CONTRIBUTING.md` contents into a web page?

Reply:
Yes